hasher-matcher-actioner/hmalib/common/s3_adapters.py

```diff
@@ -251,7 +267,13 @@ def _get_supported_indicator_types(
 
     @property
     def checkpoint_s3_key(self) -> str:
-        return f"{self.s3_te_data_folder}{self.privacy_group}.checkpoint"
+        return f"{self.s3_te_data_folder}{self.privacy_group}{self.CHECKPOINT_SUFFIX}"
+
+    def get_privacy_group_prefix(self) -> str:
```

Comment:
The '.' is pretty easy to miss here: without it you could also use this on line 270.

Reply:
I'll add more in the docstring, but this is necessary here. Imagine two privacy groups 123 and 1234. Unless we specify the . here, the `list_objects()` call later for a pg 123 would return datafiles for both 123 and 1234.

Reply:
[ignore-able] nit/to clarify: not questioning the need for the dot at all just having this helper. The subtle nature of that case means I'd almost not want this helper at all or handle it directly where the call is made. 
I don't know from reading the code that a prefix OR a suffix is going to have a '.' or a '/'